src/amp/components/Ad.tsx

```diff
@@ -107,22 +110,34 @@ interface CommercialConfig {
 }
 
 export const Ad = ({
+	isSticky,
 	adRegion,
 	edition,
 	section,
 	contentType,
 	config,
 	commercialProperties,
 }: {
+	isSticky?: boolean;
 	adRegion: AdRegion;
 	edition: Edition;
 	section: string;
 	contentType: string;
 	config: CommercialConfig;
 	commercialProperties: CommercialProperties;
 }) => {
-	const { width, height } = sizes[0]; // Set initial size to MPU
-	const multiSizes = sizes.map((e) => `${e.width}x${e.height}`).join(',');
+	let width;
+	let height;
+	let multiSizes;
+
+	if (isSticky) {
+		[{ width, height }] = stickySizes; // Set initial size to Mobile Leaderboard
+		multiSizes = stickySizes.map((e) => `${e.width}x${e.height}`).join(',');
+	} else {
+		[{ width, height }] = sizes; // Set initial size to MPU
+		multiSizes = sizes.map((e) => `${e.width}x${e.height}`).join(',');
+	}
+
```

Comment:
```suggestion
	const { width, height } = isSticky ? stickySizes : sizes[0]; // Set initial size to MPU
	const multiSizes = sizes.map((e) => `${e.width}x${e.height}`).join(',');
```

Reply:
Hey, thanks for the suggestion. It doesn't cover using `stickySizes` vs `sizes` for multiSizes though. I was contemplating doing something like this, which has a bit less repetition in it:
```js
const adSizes = isSticky ? stickySizes : sizes;
[{ width, height }] = adSizes; // Set initial size as first element
multiSizes = adSizes.map((e) => `${e.width}x${e.height}`).join(',');
```